### PR TITLE
add some class documentation for controllers

### PR DESCRIPTION
**Pull request by johrstrom**
add some class documentation for controllers

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1202821104799596/1202956026999733) by [Unito](https://www.unito.io)

### CODE REVIEW

File: apps/dashboard/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# The parent controller for all other controllers.
```
Comment (Oglopf) [inline apps/dashboard/app/controllers/application_controller.rb:1]:
```suggestion
# One controller to rule them all, and in the view and model, bind them.
```
This is the dumbest comment I've ever left and will see myself out.

**Review by Oglopf — APPROVED** [review]
Looks good to me!

File: apps/dashboard/app/controllers/application_controller.rb (hunk 1 above)
Comment (johrstrom) [reply apps/dashboard/app/controllers/application_controller.rb:1]: LOL

File: apps/dashboard/app/controllers/transfers_controller.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 require "rclone_util"
 
+# The controller for transer pages /dashboard/transfers
```
Comment (gerald-byrket) [inline apps/dashboard/app/controllers/transfers_controller.rb:3]:
```suggestion
# The controller for transfer pages /dashboard/transfers
```

**Review by gerald-byrket — COMMENTED** [review]
Once the suggestion is committed, then this is good to go.

**Review by gerald-byrket — APPROVED** [review]
Approved once test failure starts passing.

**Pull request merged**